yt_dlp/extractor/wrestleuniverse.py

```diff
@@ -126,10 +126,11 @@ def _real_extract(self, url):
 
         return {
             'id': video_id,
+            'title': '',
```

Comment:
Do you have a test case for this?

Reply:
I don't have any examples of the API response or nextJS JSON not having a `displayName` field, no. My original thought behind doing this was so that the extractor would always return an info_dict with a `title` field, so that the user could continue to download the video even if the API metadata changed. But I guess it is instead better to raise the error so any problem/change is brought to our attention and can be fixed. I'll remove these

Reply:
1. title='' => Auto-generate title
2. title=None => Issue warning and auto-generate title
3. title missing => Raise error

I believe what you want here is (2). Though, we could probably change the behavior of (3) in core code if that'd make things easier. The error for missing title is really just vestigial

Reply:
> we could probably change the behavior of (3) in core code 

I like this idea (but am also OK with 2)

This would suffice, right? 
```diff
diff --git a/yt_dlp/YoutubeDL.py b/yt_dlp/YoutubeDL.py
index fd280726f..676201705 100644
--- a/yt_dlp/YoutubeDL.py
+++ b/yt_dlp/YoutubeDL.py
@@ -2411,10 +2411,7 @@ def check_thumbnails(thumbnails):
     def _fill_common_fields(self, info_dict, final=True):
         # TODO: move sanitization here
         if final:
-            title = info_dict.get('title', NO_DEFAULT)
-            if title is NO_DEFAULT:
-                raise ExtractorError('Missing "title" field in extractor result',
-                                     video_id=info_dict['id'], ie=info_dict['extractor'])
+            title = info_dict.get('title')
             info_dict['fulltitle'] = title
             if not title:
                 if title == '':
```